lib/codelyzer/5.1.x/tslint.json

```diff
@@ -0,0 +1,55 @@
+{
+  "extends": ["codelyzer"],
+  "rules": {
+    // Codelyzer recommended rules
+    "directive-class-suffix": [true, "Directive"],
+    "directive-selector": [true, "attribute", "", "camelCase"],
+    "component-class-suffix": [true, "Component"],
+    "component-max-inline-declarations": true,
+    "component-selector": [true, "element", "", "kebab-case"],
+    "no-inputs-metadata-property": true,
+    "no-outputs-metadata-property": true,
+    "no-host-metadata-property": true,
+    "no-attribute-decorator": true,
+    "no-conflicting-lifecycle": true,
+    "no-input-rename": true,
+    "no-output-native": true,
+    "no-output-on-prefix": true,
+    "no-output-rename": true,
+    "no-forward-ref": true,
+    "no-lifecycle-call": true,
+    "no-queries-metadata-property": true,
+    "prefer-output-readonly": true,
+    "use-component-selector": true,
+    "use-lifecycle-interface": true,
+    "use-pipe-transform-interface": true,
+
+    // Codelyzer rules
+    "contextual-decorator": true,
+    "contextual-lifecycle": true,
+    "no-pipe-impure": true,
+    "no-unused-css": true,
+    "prefer-on-push-component-change-detection": false,
+    "relative-url-prefix": true,
+    "template-accessibility-alt-text": true,
+    "template-accessibility-elements-content": true,
+    "template-accessibility-label-for": true,
+    "template-accessibility-tabindex-no-positive": true,
+    "template-accessibility-table-scope": true,
+    "template-accessibility-valid-aria": true,
+    "template-banana-in-box": true,
+    "template-click-events-have-key-events": true,
+    "template-conditional-complexity": [true, 4],
+    "template-cyclomatic-complexity": [true, 10],
+    "template-mouse-events-have-key-events": true,
+    "template-no-any": true,
+    "template-no-autofocus": true,
+    "template-no-call-expression": false,
+    "template-no-distracting-elements": true,
+    "template-no-negated-async": true,
+    "template-use-track-by-function": true,
+    "use-component-view-encapsulation": true,
+    "use-injectable-provided-in": false,
```

Comment:
I think we should remove it if is disabled anyway right? Same for `5.2.x` config ;)